app/invocation/invocation.css

```diff
@@ -530,6 +530,41 @@
   margin-right: 10px;
 }
 
+#cache-miss-popup {
+  right: auto;
+  flex-direction: column;
+  padding: 12px;
+}
+
+#cache-miss-popup .title {
+  font-weight: 600;
+  font-size: 15px;
+  background: white;
+  padding: 4px;
+}
+
+#cache-miss-popup button {
+  margin-top: 5px;
+}
+
+.debug-cache-miss-container {
+  position: relative;
+}
+
+.debug-cache-miss-container button {
+  font-size: 14px;
+}
+
+.debug-cache-miss-button {
+  padding-left: 5px;
+}
+
+.checkbox-row {
```

Comment:
nit: make this selector more specific like `.debug-cache-miss-blah .checkbox-row`